truffleruby/src/main/ruby/core/truffle/ffi/ffi.rb

```diff
@@ -76,6 +76,9 @@ def find_type(name)
     # Given a +type+ as a number, indicate how many bytes that type
     # takes up on this platform.
 
+    Rubinius::NativeFunction = Class.new
+    Rubinius::FFI::Enum = Class.new
```

Comment:
These two added just to satisfy a "missing constant" error in the following method.